examples/register_and_change_username.php

```diff
@@ -43,35 +41,17 @@
     ],
 ];
 
-$dispatch = Kernel\buildCommandDispatcher(
-    $commandMap,
-    $factories['eventStore'],
-    $factories['snapshotStore']
-);
+$dispatch = Kernel\buildCommandDispatcher($factories['eventStore']())($factories['snapshotStore']())($commandMap);
```

Comment:
you would call that step with "null", but you're right, that give no advantages here, so I will update this again.